mne/io/eeglab/eeglab.py

```diff
@@ -0,0 +1,461 @@
+            loc_z = _to_loc(chanloc.Z)
+            locs = np.r_[-loc_y, loc_x, loc_z]
+            if np.unique(locs).size == 1:
+                locs_available = False
+            pos.append(locs)
+        if locs_available:
+            montage = Montage(np.array(pos), ch_names, kind, selection)
+    elif isinstance(montage, string_types):
+        path = op.dirname(montage)
+
+    if montage is None:
+        info = create_info(ch_names, eeg.srate, ch_types='eeg')
+    else:
+        _check_update_montage(info, montage, path=path,
+                              update_ch_names=True)
+
+    info['buffer_size_sec'] = 1.  # reasonable default
+    # update the info dict
+
+    if eog == 'auto':
+        eog = [ch for ch in ch_names if ch.startswith('EOG')]
+
+    cal = 1e-6
+    for ch in info['chs']:
+        ch['cal'] = cal
+        if ch['ch_name'] in eog:
+            ch['coil_type'] = FIFF.FIFFV_COIL_NONE
+            ch['kind'] = FIFF.FIFFV_MISC_CH
+
+    return info
+
+
+def read_raw_eeglab(input_fname, montage=None, preload=False, eog=(),
+                    verbose=None):
+    """Read an EEGLAB .set file
+
+    Parameters
+    ----------
+    input_fname : str
+        Path to the .set file. If the data is stored in a separate .fdt file,
+        it is expected to be in the same folder as the .set file.
+    montage : str | None | instance of montage
+        Path or instance of montage containing electrode positions.
+        If None, sensor locations are (0,0,0). See the documentation of
+        :func:`mne.channels.read_montage` for more information.
+    preload : bool or str (default False)
+        Preload data into memory for data manipulation and faster indexing.
+        If True, the data will be preloaded into memory (fast, requires
+        large amount of memory). If preload is a string, preload is the
+        file name of a memory-mapped file which is used to store the data
+        on the hard drive (slower, requires less memory). Note that
+        preload=False will be effective only if the data is stored in a
+        separate binary file.
+    eog : list | tuple | 'auto'
+        Names of channels or list of indices that should be designated
+        EOG channels. If 'auto', the channel names beginning with
+        ``EOG`` are used. Defaults to empty tuple.
+    verbose : bool, str, int, or None
+        If not None, override default verbose level (see mne.verbose).
+
+    Returns
+    -------
+    raw : Instance of RawEEGLAB
+        A Raw object containing EEGLAB .set data.
+
+    Notes
+    -----
+    .. versionadded:: 0.11.0
+
+    See Also
+    --------
+    mne.io.Raw : Documentation of attribute and methods.
+    """
+    return RawEEGLAB(input_fname=input_fname, montage=montage, preload=preload,
+                     verbose=verbose)
+
+
+def read_epochs_eeglab(input_fname, events=None, event_id=None, montage=None,
+                       eog=(), add_eeg_ref=True, verbose=None):
```

Comment:
you can get rid of the `add_eeg_ref` argument here now, just pass it as `add_eeg_ref=False` to the `_BaseEpochs` constructor. From talking to @agramfort he agreed to get rid of it if possible next release.